Week-3/q1/public/js/index.js

```diff
@@ -0,0 +1,40 @@
+//Given reducer method:
+function add(a, b) {
+	return a + b;
+}
+
+// Create a memoize function
+const memoize = (fn) => {
+	const cache = new Map();
+	return function (...args) {
+		let key = args.toString();
```

Comment:
toString is a good option , but it doesn't help if the order of arguments is different , however it is also not useful if the input type is object.